src/dune_rules/lib_info.ml

```diff
@@ -369,7 +369,6 @@ let kind t = t.kind
 let default_implementation t = t.default_implementation
 let obj_dir t = t.obj_dir
 let virtual_ t = t.virtual_
-let is_parameter t = t.kind = Parameter
```

Comment:
I am not in favor of this change, since this predicate is already used in a helpful way in a pending PR, I don't think it does any harm. Bub also, we are currently editing this file in two substantial PRs right now, so this trivial change will probably introduce merge conflicts. Even if we want to remove it, doing it like this doesn't seem to help anything.